Gemfile

```diff
@@ -45,7 +45,9 @@ gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 gem "bootsnap", require: false
 
 # Use Sass to process CSS
-# gem "sassc-rails"
+gem "sassc-rails"
```

Comment:
Noting my understand of the current asset pipeline is incomplete and evolving... I'm surprised to see both `sassc-rails` and `cssbundling-rails` in here. I thought the latter would handle sass compilation by itself, like we're doing in hydra_etd now.